### DISCUSSION TITLE
HA media browser wouldn't play clips

### DISCUSSION THREAD

**Discussion by B0ndo2** [blakeblackshear/frigate#1678] [Ask A Question]
I have an old MJPEG camera which am using with Frigate and detection works fine and clips are saved. When I try to play the clips from the media browser of HA on Firefox I get a message saying "No video with supported format and MIME found". If I access the video using SMB, it plays fine with VLC. The same problem is with the Android app (no playback but without an error message)

**Comment by HomeAssistbs** [comment]
I have the same issue. Did you ever find a solution?
I have Frigate running in Home Assistant but when I attempt to playback recordings via the HA GUI on my PC, I get "No video with supported format and MIME type found."
If however I download a recoding  to my PC then it plays back. There I appear to have the necessary codec on my PC.
Also I can view recordings via the HA GUI on my android phone.
Could someone please advise what is wrong and how I can get to view the recordings in HA on my PC.
Thanks in advance
